src/Generation/FieldDetails.php

```diff
@@ -236,6 +243,15 @@ private function determineIsRequired(DescriptorProto $containingMessage, FieldDe
         return $isRequired;
     }
 
+    private function isFieldInfoUuid(FieldDescriptorProto $field)
```

Comment:
Nit: I'd name this `isFormatUuid4` - that's the specific field that contains this, a little more specific.

Reply:
Makes sense. I've updated it.